.github/workflows/generate-tpch-s3.yml

```diff
@@ -37,6 +32,4 @@ jobs:
         run: |
           # We run each query once to make sure we don't upload a file if there's a bug that causes a panic.
           cargo run --release --bin tpch -- --formats parquet,vortex -i1
-          aws s3 cp --recursive bench-vortex/data/tpch/1 $REMOTE_PATH
-          # We can now clear generated data
-          rm -rf bench-vortex/data/
+          aws s3 cp --recursive bench-vortex/data/tpch/1 s3://vortex-bench-dev-eu/tpch-sf1/
```

Comment:
AFAIK this will only delete old files iff we have new file with exact same name. Probably true but... maybe a source of bugs in the future?

Reply:
I think we will need a more elaborate setup in the future. This is good enough for now. We need to make a proper bench crate that will sync files with expected state